generator/autogenlist.ts

```diff
@@ -8,8 +8,224 @@ import { postProcessor as policyProcessor } from './processors/Microsoft.Authori
 import { postProcessor as securityInsightsPostProcessor } from './processors/Microsoft.SecurityInsights';
 import { lowerCaseEquals } from './utils';
 
+// Going forwards, providers are onboarded by default.
+// The providers listed here are the only ones **not** onboarded.
+const disabledProviders: AutoGenConfig[] = [
```

Comment:
>disabledProviders [](http://example.com/codeflow?start=6&length=17)

What is the difference between this and blocklist?

Reply:
Good question. I see the blocklist more like "these paths should be totally ignored, and do not apply for schema gen", whereas the config here is "these RPs have not yet been onboarded, but will be in the future". Let me add some comments and possible rename variables to try and make this clearer.